### PR TITLE
Add HCL2 examples to the virtualbox builders

### PR DESCRIPTION
**Pull request by sylviamoss**
Part of https://github.com/hashicorp/packer/issues/9668

### CODE REVIEW

File: website/pages/docs/builders/virtualbox/iso.mdx [hunk 1]
```diff
@@ -230,41 +248,30 @@ uploaded is controllable by `guest_additions_path`, and defaults to
 "VBoxGuestAdditions.iso". Without an absolute path, it is uploaded to the home
 directory of the SSH user.
 
-## VBoxManage Commands
```
Comment (sylviamoss) [inline website/pages/docs/builders/virtualbox/iso.mdx:233]: This is a weird diff! I move `VBoxManage Commands` to a partial and added to the VBoxManage Configuration section at line 156 above.

File: builder/virtualbox/common/export_config.go [hunk 2]
```diff
@@ -31,6 +32,19 @@ type ExportConfig struct {
...
+	// ```hcl
+	//	source "virtualbox-iso" "basic-example" {
+	//		export_opts = [
+	//	      "--manifest",
+	//	      "--vsys", "0",
+	//	      "--description", "{{user `vm_description`}}",
+	//	      "--version", "{{user `vm_version`}}"
+	//	   	]
+	//		format = "ova"
+	//  }
```
Comment (azr) [inline builder/virtualbox/common/export_config.go:45]: Not sure if this is github but it looks like the indentation is a bit off:
```suggestion
	//		export_opts = [
	//	          "--manifest",
	//	          "--vsys", "0",
	//	          "--description", "{{user `vm_description`}}",
	//	          "--version", "{{user `vm_version`}}"
	//	   	]
	//		format = "ova"
	//  }
```

File: builder/virtualbox/common/vboxmanage_config.go [hunk 3]
```diff
@@ -6,17 +6,40 @@ import (
...
-	// the virtual machine being created. The value of this is an array of
-	// commands to execute. The commands are executed in the order defined in
-	// the template. For each command, the command is defined itself as an
-	// array of strings, where each string represents a single argument on the
-	// command-line to `VBoxManage` (but excluding `VBoxManage` itself). Each
-	// arg is treated as a [configuration
-	// template](/docs/templates/engine), where the `Name` variable is
-	// replaced with the VM name. More details on how to use `VBoxManage` are
-	// below.
+	// the virtual machine being created. An example is shown below that sets the memory and number of CPUs
```
Comment (azr) [inline builder/virtualbox/common/vboxmanage_config.go:16]:
```suggestion
	// the virtual machine being created. The examples shown below set the memory and number of CPUs
```

File: builder/virtualbox/common/vboxmanage_config.go [hunk 4]
```diff
@@ -6,17 +6,40 @@ import (
...
+	//	  ["modifyvm", "{{.Name}}", "--cpus", "2"]
+	// ]
+	// ```
+	//
+	// In HCL2:
+	// ```hcl
+	// vboxmanage = [
+	//    ["modifyvm", "{{.Name}}", "--memory", "1024"],
+	//	  ["modifyvm", "{{.Name}}", "--cpus", "2"]
+	// ]
```
Comment (azr) [inline builder/virtualbox/common/vboxmanage_config.go:32]:
```suggestion
	// vboxmanage = [
	//    ["modifyvm", "{{.Name}}", "--memory", "1024"],
	//    ["modifyvm", "{{.Name}}", "--cpus", "2"]
	// ]
```

**Review by azr — APPROVED** [review]
Nice !

**Pull request merged**